src/modules/comment.js

```diff
@@ -1,16 +1,32 @@
+const popUp = document.querySelector('.comment-popup');
 export default class Comment {
   static createCommentPopUp = (data) => {
     const newArticle = document.createElement('article');
     newArticle.innerHTML = `<div class= "flex"> 
-  <img src= ${data.url} alt="Image"/> 
+  <img src= ${data.artworkUrl100} alt="Image"/> 
   <span class='material-symbols-outlined'>close</span>
   </div>
-  <h2>Space ${data.count}</h2>
+  <h2>Space ${data.index}</h2>
   <div class= "pop-details">
     <p>Fuel titatium</p>
     <p>Length: 10000</p>
     <p>Weight: 10000</p>
     <p>Power: 10000</p>
   </div>`;
```

Comment:
- [ ] One of the [requirements ](https://github.com/idungstanley/API-based-webAPP/issues/14#issue-1237792105) for this task is to display specific information about the card. I think we could use the ID to request particular information and then display it. Example: Album number of tracks, collection Price, country, primary gender.

Reply:
Done